### PR TITLE
Added PDF preview feature in workflow attachments

### PR DESCRIPTION
**Pull request by tanya17-05**
Added PDF preview as mentioned in [issue#2](https://github.com/elixir-cloud-aai/krini/issues/2) in workflow attachment field

### CODE REVIEW

File: src/components/RunWorkflow.js [hunk 1]
```diff
@@ -344,24 +345,32 @@ const RunWorkflow = ({ isLoggedIn, showToast }) => {
...
-                  >
-                    <path
-                      strokeLinecap="round"
-                      strokeLinejoin="round"
-                      d="M7 21h10a2 2 0 002-2V9.414a1 1 0 00-.293-.707l-5.414-5.414A1 1 0 0012.586 3H7a2 2 0 00-2 2v14a2 2 0 002 2z"
-                    />
-                  </svg>
-                )}
+                {(() => {
+                  console.log(file.preview.type)
```
Comment (git-anurag-hub) [inline src/components/RunWorkflow.js:349]: Remove consoles!

**Review by git-anurag-hub — CHANGES_REQUESTED** [review]
@tanya17-05 Looks good to me. Can you see the little change I suggested!

**Comment by git-anurag-hub** [comment]
Also, @tanya17-05 Can you post the screenshot of the demo images & different types of files as file attachments. It will be great to see the changes before merging the changes.

File: src/components/RunWorkflow.js (hunk 1 above)
Comment (tanya17-05) [reply src/components/RunWorkflow.js]: Done @git-anurag-hub

**Review by git-anurag-hub — APPROVED** [review]

**Pull request merged**